Model/Capabilities.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Omise\Payment\Model;
+
+use Omise\Payment\Model\Api\Capabilities as OmiseCapabilitiesAPI;
+use Omise\Payment\Model\Omise;
+
+class Capabilities
```

Comment:
Again, should this really be called `capabilities`?

Reply:
I think so.. it is about getting capabilities.